src/css/components/files/_c-cart-card-body.scss

```diff
@@ -0,0 +1,5 @@
+.c-cart-card-body{
+  width: 100%;
```

Comment:
`display:block` and `width:100%' not needed, since `div`s have these by default.